src/components/Reports/ReportsPartialsCampaigns/ReportsPartialsCampaigns.js

```diff
@@ -67,105 +68,139 @@ const ReportsPartialsCampaigns = ({ location, dependencies: { dopplerApiClient }
                   <h3>
                     <FormattedMessage id="reports_partials_campaigns.header_title" />
                   </h3>
+                  <S.NameCampign>
```

Comment:
is ok that? i think the names reference a type of the object. You remember @cbernat ?

Reply:
In order to have real meaning and make this component reusable, perhaps renaming this component into `MainReportBox` or `MainReportContainer` or something that gives more semantic to the html, what do you this @GusBaamonde ?